.gitignore

```diff
@@ -1,3 +1,4 @@
 .vagrant
 vagrant_ansible_inventory_default
 tests.pyc
+secret
```

Comment:
I think the idea here is that you're free to ``git add -f`` so your
secrets are tracked but the sensible default is to prevent
them from being added by accident.

On 03/09, Carl Meyer wrote:
> > @@ -1,3 +1,4 @@
> >  .vagrant
> >  vagrant_ansible_inventory_default
> >  tests.pyc
> > +secret
> 
> I wouldn't personally want this directory git-ignored. I consider an Ansible deploy repo (as opposed to a reusable role) to be private, and I treat it that way. I want my important secrets tracked in git, not subject to deletion by a stray `git clean`.
> 
> But if others prefer to have it git-ignored, I can just revert that change in my private fork for my server.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sovereign/sovereign/pull/526/files#r55609987